dev-tools/e2e/kibana.yml

```diff
@@ -0,0 +1,33 @@
+server.name: kibana
+server.host: "0.0.0.0"
+server.ssl.enabled: false
+elasticsearch.hosts: [ "http://elasticsearch:9200" ]
+elasticsearch.serviceAccountToken: "${KIBANA_TOKEN}"
+
+xpack.fleet.agents.enabled: true
+xpack.fleet.agents.fleet_server.hosts: ["http://fleet-server:8220"]
```

Comment:
We may want to include  variants for `localhost` as well as `https` entries to cover all test cases

Reply:
+1 on adding an `https` entry here to make it closer to a real life scenario and what we propose to the user (maybe worth creating a follow up issue)